site/reference/fn/doc/README.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Doc"
+linkTitle: "doc"
+type: docs
+description: >
+    Get the help text from a function image
+---
+
+<!--mdtogo:Short
+    Display the documentation for a function
+-->
+
+### Examples
+
+<!--mdtogo:Examples-->
+
+```shell
+# diplay the documentation for image gcr.io/kpt-fn/set-namespace:v0.1.1
+kpt fn doc --image gcr.io/kpt-fn/starlark:unstable
+```
+
+<!--mdtogo-->
+
+### Synopsis
```

Comment:
It's fine to say how the function works under the covers in the detail section here.